exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/logs/LogMarshaler.java

```diff
@@ -50,8 +50,7 @@ static LogMarshaler create(io.opentelemetry.sdk.logs.data.LogData logData) {
         MarshalerUtil.toBytes(logData.getSeverityText()),
         anyValueMarshaler,
         attributeMarshalers,
-        // TODO (trask) implement droppedAttributesCount in LogRecord
-        0,
+        logData.getTotalAttributeCount() - logData.getAttributes().size(),
```

Comment:
Would it make sense to encapsulate this behind `logData.getDroppedAttributesCount()` (new method)?

Reply:
If this was greenfield I'd go with the dropped attribute count, but this is symmetric with [SpanData#getTotalAttributeCount()](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/trace/src/main/java/io/opentelemetry/sdk/trace/data/SpanData.java#L151).